src/transformStateWithClones.js

```diff
@@ -7,7 +7,37 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let copy = { ...state };
+
+  actions.forEach(elem => {
+    if (result.length !== 0) {
+      copy = { ...result[result.length - 1] };
+    }
```

Comment:
why do you need this? remove

Reply:
Each element of the resulting array has to represent the state produced by the next operation. Otherwise it will fail.